bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -311,6 +311,60 @@ def rotate_gripper_with_delta(self, wrist_yaw=0.0, wrist_roll=0.0):
         self.set_arm_joint_positions(new_arm_joint_states)
         time.sleep(0.5)
 
+    def move_gripper_to_points(
```

Comment:
this is the duplication to `move_gripper_to_point`

Reply:
this is fixed, but we need this function